phlop/procs/parallel_processor.py

```
@@ -246,17 +286,41 @@ def cancel_running():
 
     while running or pending:
         try:
-            job_idx, value, error = queue.get(timeout=TIMEOUT)
+            job_idx, value, error = queue.get(timeout=queue_timeout)
         except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)